Shared/Mirror/spawnMirror.js

```diff
@@ -0,0 +1,72 @@
+// JavaScript source code
```

Comment:
Missing file headers with copyright notification, creator and optional short description of the script, see other scripts in repo for examples.